app/src/docs/v1.api-spec.yaml

```diff
@@ -762,6 +793,38 @@ paths:
           $ref: "#/components/responses/Forbidden"
         default:
           $ref: "#/components/responses/Error"
+  /object/{objectId}/sync:
+    get:
+      summary: Synchronize an existing object
+      description: |-
+        Synchronize an existing COMS object so that any changes made externally (e.g. using an S3 client
+        to upload a new version) are tracked by COMS.
+
+        The object is added to an internal processing queue; once it is eventually processed,
+        the changes are tracked by COMS.
+      tags:
+        - Sync
+      operationId: syncObject
+      parameters:
+        - $ref: "#/components/parameters/Path-ObjectId"
+        - $ref: "#/components/parameters/Query-Public"
+      responses:
+        "202":
+          description: Returns the number of objects that have been added to the queue.
+          content:
+           text/plain:
+              schema:
```

Comment:
Incorrect indentation spacing. Make sure you are right click formatting with VSCode.